templates/Web/Projects/AddPackageJsonDefault/.template.config/template.json

```diff
@@ -10,7 +10,7 @@
     "type": "project",
     "wts.type": "project",
     "wts.projecttype": "all",
-    "wts.frontendframework": "ReactJS|AngularJS",
+    "wts.frontendframework": "ReactJS|AngularJS|Vue",
```

Comment:
Note this says AngularJS here, but this will be changed to "Angular" once Jason's PR is in.

This allows the Base default project to be merged in for React Angular and Vue selections.